aea/test_tools/docker_image.py

```diff
@@ -130,7 +124,7 @@ def launch_image(
     if not success:
         container.stop()
         container.remove()
-        pytest.fail(f"{image.tag} doesn't work. Exiting...")
+        raise DockerException(f"{image.tag} doesn't work. Exiting...")
```

Comment:
can never be covered by pytest without test failure

Reply:
No issues, just add a no cover tag.

Reply:
zero pragmatism